routers/web/web.go

```diff
@@ -813,7 +813,7 @@ func registerRoutes(m *web.Route) {
 						}, reqPackageAccess(perm.AccessModeWrite))
 					})
 				})
-			}, ignSignIn, context.PackageAssignment(), reqPackageAccess(perm.AccessModeRead))
+			}, context.PackageAssignment(), reqPackageAccess(perm.AccessModeRead))
```

Comment:
If `RequireSignInView` is true, users should download files after login? So why we should remove this middleware?